### PR TITLE
feat/starships

### PR DESCRIPTION
**Pull request by soyfriss**
- Add starships folder inside components.
- Create Starships component.
- Add Starships to render on /starships route.

### CODE REVIEW

File: arturito/src/components/StarshipsSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        dataIndex: 'manufacturer',
+        key: 'manufacturer'
+    },
+    {
+        title: 'Passengers',
+        dataIndex: 'passengers',
+        key: 'passengers'
+    },
+    {
+        title: 'Cantidad de films',
```
Comment (DiegoFormoso) [inline arturito/src/components/StarshipsSection/index.tsx]: Fijate que aca se te mezclo castellano con ingles, estaria bueno que sigas en ingrles todos los title

**Review by DiegoFormoso — CHANGES_REQUESTED** [review]

File: arturito/src/components/StarshipsSection/index.tsx (hunk 1 above)
Comment (soyfriss) [reply arturito/src/components/StarshipsSection/index.tsx]: OK Diego, lo corrijo!

**Review by DiegoFormoso — DISMISSED** [review]
it's ok

**Review by DiegoFormoso — APPROVED** [review]
it's ok

**Pull request merged**